src/resources/boot.lua

```diff
@@ -149,8 +149,12 @@ function lovr.run()
   return function()
     lovr.event.pump()
     for name, a, b, c, d in lovr.event.poll() do
-      if name == 'quit' and (not lovr.quit or not lovr.quit()) then
-        return a or 0, b
+      if name == 'restart' then
+        local cookie = lovr.restart and lovr.restart()
+        return 'restart', cookie
+      end
+      if name == 'quit' and (not lovr.quit or not lovr.quit(a)) then
```

Comment:
Should this be `elseif`?  idk, not important

Reply:
They are logically equivalent but the elseif reads better.